public/scripts/wiki.js

```diff
@@ -50,7 +52,6 @@ var WikiBox = React.createClass({
 
     return (
       <div className="wiki-box">
-        <Navbar />
```

Comment:
why move navbar from wiki-box to article box? in this case, we should just get rid of WikiBox since ArticleBox is doing the same thing